Big-Spender/readme.md

```diff
@@ -132,6 +171,16 @@ INSERT YOUR QUERIES HERE
 
 **You:** No problem, glad I could help you out.
 
+select id from suppliers where supplier = 'Dell';
+insert into suppliers(supplier) values ('Dell');
+select id from expense_types where expense_type = 'Hardware';
+insert into expense_types (expense_type) values ('Hardware');
+select id from expense_areas where expense_area = 'IT';
+insert into expense_areas (expense_area) values ('IT');
+
```

Comment:
Great attention to detail! The 'Hardware', 'Dell', and 'IT' rows were missing from the other tables, and you've done well by adding the necessary insert statements to populate them.